producer.go

```diff
@@ -295,6 +332,51 @@ type insertPayload struct {
 	Queue string `json:"queue"`
 }
 
+func (p *producer) handleJobControlNotification(workCtx context.Context) func(notifier.NotificationTopic, string) {
```

Comment:
Okay, so if I understand correctly, the "job control" topic is shared between (1) job cancel notifications, and (2) queue pause/resume notifications.

It's not super clear to me what these have in common, except that they're both topics used by the producer. But job inserts are also topics shared by the producer, so by that logic, shouldn't they also share the same topic?

Reply:
The topic naming is not ideal for this. We could definitely keep the actual _job_ control topic separate from these messages which are more of _queue_ control; the only downsides to that are another constant and another subscription that must be initialized when each producer starts.

Another option is to pick a new dual purpose name.

Reply:
I removed the "job" bit from the naming and it's now just "control" throughout. Let me know if you want to paint the shed another color before merge 🎨 🖌️

Reply:
Heh heh. Can't have enough layers of paint! LG.